node/resolvers/checkout.ts

```diff
@@ -0,0 +1,16 @@
+export const nearestPickupPoints = async (
+  _: any,
+  { lat, long }: CheckoutPickupPointsArgs,
+  ctx: Context
+) => {
```

Comment:
It would be great to type what we are returning here!
Something like [this line](https://github.com/vtex-apps/cross-device-cart/blob/V1/node/resolvers/replaceCart.ts#L15)